ui/job-view/headerbars/SecondaryNavBar.jsx

```diff
@@ -345,6 +349,8 @@ class SecondaryNavBar extends React.PureComponent {
               <span
                 className="btn btn-view-nav btn-sm"
                 onClick={toggleFieldFilterVisible}
+                role="button"
+                tabIndex="0"
```

Comment:
This is fine, but we should prefer using semantic elements instead of spans and divs. Using the Button component from react-strap will provide the role attribute so I'd suggest using that here (and where ever else it makes sense).